packages/plugin-ext/src/main/browser/view/tree-views-main.ts

```diff
@@ -108,6 +108,28 @@ export class TreeViewsMainImpl implements TreeViewsMain, Disposable {
         }
     }
 
+    async $setMessage(treeViewId: string, message: string): Promise<void> {
+        const viewPanel = await this.viewRegistry.getView(treeViewId);
+        const widget = viewPanel && viewPanel.widgets[0];
+
+        if (widget instanceof TreeViewWidget) {
+            widget.message = message;
+        } else {
```

Comment:
i'm not sure about logging errors in else cases, other methods don't do it. It could be that message is resolved by the time when a user already closed the tree, there is nothing bogus to ignore message then.

Reply:
It was requested by @tsmaeder. I also think the logging is not needed here. 
During the run this error appears several times, while not always indicate a real problem with absent widget

Reply:
So what we're trying to guard here is the widget being present, not that it might be there but not a TreeWidget? Then why not do a `if (widget)` It has the same effect, but IMO would be better express the meaning.

Reply:
If we like here https://github.com/eclipse-theia/theia/pull/7055#discussion_r381852705 then we should log error if only metadata is missing. The same for title.